### PR TITLE
fix: correct links

### PR DESCRIPTION
**Pull request by tdstein**

### CODE REVIEW

File: 14-mlops/03-promoting-content-from-staging-to-production/index.qmd [hunk 1]
```diff
@@ -40,7 +40,8 @@ source_content = source_client.content.get(SOURCE_CONTENT_GUID)
 target_content = target_client.content.get(TARGET_CONTENT_GUID)
 ```
 
-Next, obtain the bundle you want to promote using a bundle identifier (id), then download the bundle archive. See [Finding Bundles](../../07-bundles/02-finding-bundles) and [Downloading Bundles](../../07-bundles/06-downloading-bundles) for additional information.
+Next, obtain the bundle you want to promote using a bundle identifier (id), then download the bundle archive.
+<!-- See [Finding Bundles](../../07-bundles/02-finding-bundles) and [Downloading Bundles](../../07-bundles/06-downloading-bundles) for additional information. -->
```
Comment (tdstein) [inline 14-mlops/03-promoting-content-from-staging-to-production/index.qmd:44]: We need to implement this, and then we can add it back in.

**Review by toph-allen — COMMENTED** [review]
Will these have to change more now that numbers have been removed?

**Review by toph-allen — APPROVED** [review]
Nvm, I saw which PR came first.

**Comment by tdstein** [comment]
I'm just going to duplicate this work in #140

**Pull request closed without merging**